apps/openmw/mwgui/jailscreen.cpp

```diff
@@ -0,0 +1,136 @@
+#include <MyGUI_RenderManager.h>
+#include <MyGUI_ScrollBar.h>
+#include <MyGUI_Gui.h>
+#include <MyGUI_TextBox.h>
+
+#include "../mwbase/windowmanager.hpp"
+#include "../mwbase/mechanicsmanager.hpp"
+#include "../mwbase/world.hpp"
+#include "../mwbase/environment.hpp"
+
+#include "../mwmechanics/npcstats.hpp"
+
+#include "../mwworld/esmstore.hpp"
+#include "../mwworld/store.hpp"
+#include "../mwworld/class.hpp"
+
+#include "jailscreen.hpp"
+
+namespace MWGui
+{
+    JailScreen::JailScreen()
+        : WindowBase("openmw_loading_screen.layout"),
```

Comment:
We need a separate layout here, I think. This one would be slightly off-center, because the loading progress bar is at the bottom of a 300x300 screen.